### PR TITLE
Query node examples

### PR DESCRIPTION
**Pull request by metmirr**
The PR adds Kusama Query Node in examples. The query node runs against to Kusama live network.

### CODE REVIEW

File: query-node/examples/kusama-query-node/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,57 @@
+enum ProposalStatus {
```
Comment (bedeho) [inline query-node/examples/kusama-query-node/schema.graphql:1]: I think this is a fine representation, in the future we can add variants which hold the specific block number when the rejection or approval occured.

**Review by dzhelezov — APPROVED** [review]
Looks good!

File: query-node/examples/kusama-query-node/mappings/treasury/proposal.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import { Proposal } from '../../generated/graphql-server/src/modules/proposal/proposal.model';
+import { ProposalStatus } from '../../generated/graphql-server/src/modules/enums/enums';
+import { assert } from 'console';
+
+// New proposal
+export async function handleProposed(db: DB, event: SubstrateEvent) {
+  const { ProposalIndex } = event.event_params;
+  if (event.extrinsic) {
+    const proposal = new Proposal();
+    proposal.proposalIndex = ProposalIndex.toString();
```
Comment (bedeho) [inline query-node/examples/kusama-query-node/mappings/treasury/proposal.ts:11]: Seeing this mapping made me make this issue https://github.com/Joystream/joystream/issues/1024 for the future.

**Review by bedeho — APPROVED** [review]
Looks great, this is coming together nicely.

**Comment by dzhelezov** [comment]
@metmirr  I think the following env variables should be added to .env:

```
# DEFUALT GraphQL server HOST and PORT 
GRAPHQL_SERVER_PORT=4000
GRAPHQL_SERVER_HOST=localhost
WARTHOG_APP_PORT=4000
WARTHOG_APP_HOST=localhost
```

**Pull request merged**